.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -1,7 +1,7 @@
 name: Bug Report
 description: Report something that isn't working as expected.
 title: "bug: "
-labels: ["type: bug", "status: needs triage"]
+labels: [ "type: bug", "status: needs triage" ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)